libs/wire-subsystems/src/Wire/UserSubsystem.hs

```diff
@@ -53,11 +53,30 @@ instance Default UserProfileUpdate where
         supportedProtocols = Nothing
       }
 
+-- | how to get an account for a user
+data GetBy = MkGetBy
+  { -- | whether or not to include ending invitations in the lookups }
```

Comment:
```suggestion
  { -- | whether or not to include ending invitations in the lookups
```

Reply:
why no `-XStrictData` for this module - are there any types here that require laziness?